rs_gazebo/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rs_gazebo)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+  CATKIN_DEPENDS gazebo_ros gazebo_ros_control rs_description
+)
+
+install(FILES
+  launch/gazebo.launch
+  worlds/office.world
```

Comment:
which file re we running? Is it reallly office.world?

Reply:
It is a typo. Changed in  ab2d03dcc65d68858d7bc52654391fd2d3d36552